ResearchKit/Common/ORKReviewStep.md

```diff
@@ -0,0 +1,81 @@
+<h2>Introduce</h2>
+- `ORKReviewStep`
+- `ORKReviewStepViewController`
+- `ORKReviewStepViewControllerDelegate`
+- `ORKReviewStepReviewDirection`
+
+<h2>Change</h2>
+- `ORKTaskViewController`
+
+<h2>ORKReviewStep</h2>
+<h3>properties</h3>
+    NSArray *steps
+    ORKTaskResultSource *resultSource
+    ORKReviewStepReviewDirection *reviewDirection
+<h3>use cases</h3>
+- __outside of a survey = standalone__
+  - steps array is provided upon initialization
+  - review step is at the beginning
+  - step results may not be changed
+  - review direction defaults to forward
```

Comment:
The `reviewDirection` is used by `ORKReviewStepViewController` to discover the steps that need to be reviewed. Within a survey, the review step is usually displayed after the steps that need to be reviewed. To discover those, you need to go in a reverse direction and call `stepBeforeStep:withResult` repeatedly. If an app wants to do different and display a review step at the beginning, the review direction needs to be changed to forward. That way steps will be discovered by calling `stepAfterStep:withResult:` instead.

If `ORKReviewStep` is used standalone, steps won't need to be discovered. So `reviewDirection` will be ignored.